lms/djangoapps/teams/static/teams/js/views/topics.js

```diff
@@ -21,11 +22,18 @@
                 PaginatedView.prototype.initialize.call(this);
             },
 
+            createHeaderView: function () {
+                return new PagingHeader({
+                    collection: this.options.collection,
+                    srInfo: this.srInfo,
+                    showSortControls: true
+                });
+            },
+
             render: function() {
                 var self = this;
                 this.collection.refresh()
                     .done(function() {
-                        self.collection.isStale = false;
                         PaginatedView.prototype.render.call(self);
```

Comment:
I think that's correct in most cases, but we do need this for the initial rendering of the view. In that case the initial data is already present and no `sync` event is fired, so the other views don't automagically render themselves.